server/user-bootstrap/index.js

```diff
@@ -1,32 +1,32 @@
 var superagent = require( 'superagent' ),
-	cookie = require( 'cookie' ),
 	debug = require( 'debug' )( 'calypso:bootstrap' ),
 	crypto = require( 'crypto' );
 
 var config = require( 'config' ),
 	API_KEY = config( 'wpcom_calypso_rest_api_key' ),
 	userUtils = require( './shared-utils' ),
-
+	AUTH_COOKIE_NAME = 'wordpress_logged_in',
 	/**
 	* WordPress.com REST API /me endpoint.
 	*/
 	url = 'https://public-api.wordpress.com/rest/v1/me?meta=flags';
 
-module.exports = function( userCookie, callback ) {
+
+module.exports = function( authCookieValue, callback ) {
 	// create HTTP Request object
 	var req = superagent.get( url ),
-		hmac, cookies, hash;
+		hmac, hash;
+
+	if ( authCookieValue ) {
+		authCookieValue = decodeURIComponent( authCookieValue );
```

Comment:
Ah, I see. It looks like the [`cookie` lib](https://www.npmjs.com/package/cookie#decode) does the exact same thing as this.

Reply:
Yep, I had to add it, wouldn't work without it.